src/common/utils/url.ts

```diff
@@ -184,6 +184,22 @@ function cleanPathName(basePathName: string, pathName: string) {
     return pathName;
 }
 
+function isCallsPopOutURL(serverURL: URL | string, inputURL: URL | string, callID: string) {
+    if (!serverURL || !inputURL || !callID) {
+        return false;
+    }
+
+    const parsedURL = parseURL(inputURL);
+    const server = getServerInfo(serverURL);
+    if (!parsedURL || !server || (!equalUrlsIgnoringSubpath(server.url, parsedURL))) {
+        return false;
+    }
+
+    const regexRule = `^${server.subpath}[A-Za-z0-9-_]+/${CALLS_PLUGIN_ID}/expanded/${callID}$`;
```

Comment:
Ideally I would have made this static but getting the team name is a bit tricky and if we accept it from the plugin side it would be a breaking change so using a regexp for now which I think it's fine but let me know if there are concerns.

Reply:
Is there any way we can have this call `isUrlType` instead? I spent some time consolidating all of the URL checks to be based off of that one function, so if we could continue to do that that would be ideal.

Reply:
@devinbinnie PTAL